pom.xml

```diff
@@ -1472,6 +1473,45 @@
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest-maven-plugin</artifactId>
       </plugin>
+      <!-- Build the JARs-->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <!-- Configuration of the archiver -->
+          <archive>
+            <manifestEntries>
+              <mode>development</mode>
+              <url>${project.url}</url>
+            </manifestEntries>
+            <manifest>
+              <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+              <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+            </manifest>
+          </archive>
+        </configuration>
+        <executions>
+          <execution>
```

Comment:
Yeah it can be mysterious; I feel reasonably sure about how the jar plugin works. I'd rather remove the config if we don't know that it's needed, and I have never need these. I also don't re-specify the default jar execution and it's been fine. We'll know quickly of course whether it doesn't actually produce JARs! but yeah I'd trim this down to what is necessary IMHO.